src/pkg/packaging/deb/dotnet-sharedframework-debian_config.json

```diff
@@ -30,8 +30,7 @@
 
     "debian_dependencies":{
         "%HOSTFXR_DEBIAN_PACKAGE_NAME%" : {},
-        "libssl1.0.0" : {},
```

Comment:
Can you confirm that the resulting package actually has a dependency on these two libraries (I'm worried that because of how we do the portable build, it will not).  If it doesn't can you please keep them in explicitly?